src/rpc/rpctx.cpp

```diff
@@ -2525,11 +2525,13 @@ Value sigstr(const Array& params, bool fHelp) {
 	if (!GetKeyId(params[1].get_str(), keyid)) {
 		throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "to address Invalid  ");
 	}
+	Object obj;
 	CDataStream stream(vch, SER_DISK, CLIENT_VERSION);
 	std::shared_ptr<CBaseTransaction> pBaseTx;
 	stream >> pBaseTx;
-	CAccountViewCache view(*pAccountViewTip, true);
-	Object obj;
+	if (!pBaseTx.get()) {
+		return obj;
```

Comment:
obj not assigned at all

Reply:
Once the input `str` can not be decoded, it's reasonable to return a null object.

Reply:
RYI, see `Object GetTxDetailJSON(const uint256& txhash)`.

Reply:
then it'd be better to "return null" directly.

Reply:
Done